### PR TITLE
chore: detect platform os by lsb_release

### PR DESCRIPTION
**Pull request by zoumo**

### CODE REVIEW

File: scripts/worker_image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM docker:1.10.1-dind
+
+RUN apk add --no-cache git subversion ca-certificates tzdata && \
+    ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime && \
+    echo "Asia/Shanghai" > /etc/timezone
```
Comment (superxi911) [inline scripts/worker_image/Dockerfile]: new line

File: worker/Dockerfile [hunk 2]
```diff
@@ -1,12 +1,6 @@
...
-    apk add tzdata && \
-    ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime && \
-    echo "Asia/Shanghai" > /etc/timezone
+FROM cargo.caicloud.io/caicloud/cyclone-worker:1.10.1-dind
 
 COPY ./cyclone-worker /cyclone-worker
 COPY ./start.sh /start.sh
 
-CMD ["/start.sh"]
+CMD ["/start.sh"]
```
Comment (superxi911) [inline worker/Dockerfile]: new line

File: scripts/lib/common.sh [hunk 3]
```diff
@@ -2,17 +2,66 @@
...
   if selinuxenabled && [ "$(getenforce)" = "Enforcing" ]; then
     echo "Temporarily disabling selinux enforcement"
     sudo setenforce 0
     SELINUX_DISABLED=1
   fi
 }
+
+if command_exists lsb_release ; then
+    os="$(lsb_release -si)"
+fi
```
Comment (xiaomuqiao) [inline scripts/lib/common.sh]: delete these, it is the same as line 26.

**Comment by xiaomuqiao** [comment]
LGTM

**Pull request merged**